src/twisted/conch/test/test_window.py

```diff
@@ -66,3 +73,129 @@ def test_parent(self) -> None:
         scrolled = ScrolledArea(widget)
         self.assertIs(widget.parent, scrolled._viewport)
         self.assertIs(scrolled._viewport.parent, scrolled)
+
+
+class SelectionTests(TestCase):
+    """
+    Change focused entry in L{Selection} using function keys.
+    """
+
+    def setUp(self) -> None:
+        """
+        Create L{ScrolledArea} widget with 10 elements and position selection to 5th element.
+        """
+        seq: list[bytes] = [f"{_num}".encode("ascii") for _num in range(10)]
+        self.widget = Selection(seq, None)
+        self.widget.height = 10  # type: ignore[assignment]
+        self.widget.focusedIndex = 5
+
+    def test_selection_down_arrow(self) -> None:
```

Comment:
This is a violation of the Twisted test naming convention, which is `test_selectionDownArrow`. I realize that this departs from PEP-8 so it's a common mistake :)